lotus/templates/task_instructions.py

```diff
@@ -203,18 +203,28 @@ def map_formatter(
     return messages
 
 
-def extract_formatter(df_text: str, user_instruction: str) -> list[dict[str, str]]:
+def extract_formatter(df_text: str, columns: list[str], extract_quotes: bool = True) -> list[dict[str, str]]:
+    if extract_quotes:
+        quote_fields = [f"{col} extracted quote" for col in columns]
+        all_fields = columns + quote_fields
+    else:
+        all_fields = columns
+
+    fields_str = ", ".join(all_fields)
+
     sys_instruction = (
-        "The user will provide an instruction and some relevant context.\n"
-        "Your job is to extract the information requested in the instruction.\n"
-        "Write the response in JSONL format in a single line with the following fields:\n"
-        """{"answer": "your answer", "quotes": "quote from context supporting your answer"}"""
+        "The user will provide the columns that need to be extracted and some relevant context.\n"
```

Comment:
We can punt this to a future PR, but I think that we should use [json_mode](https://docs.litellm.ai/docs/completion/json_mode) instead of prompting the model this way. This can also apply to other operators as well, so let's leave it for a future refactor.